FinTExWebSite/public/map.html

```diff
@@ -0,0 +1,474 @@
+<!DOCTYPE html>
```

Comment:
This is the template for the map page. It is a separate `.html` file instead of an Angular template in anticipation of us removing Angular from the app.

To continue removing Angular, we'd have to move our templates into a public folder so that they can be rendered without using Angular's ui-router.